boards/arm/lpcxpresso55s36/lpcxpresso55s36.dts

```diff
@@ -168,3 +170,15 @@ zephyr_udc0: &usbfs {
 	pinctrl-0 = <&pinmux_usbfs>;
 	pinctrl-names = "default";
 };
+
+&dma0 {
+	dma-channels = <52>;
+	nxp,dma-num-of-otrigs = <4>;
```

Comment:
number of otrigs should be set at SOC level

Reply:
These are now in the SOC levels